content/docs/terms/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Terminology
+menu: main
+collapseChildren: true
```

Comment:
What does this front-matter tag do?

Reply:
I have no idea - I was cargo-culting / copy-pasting from another main section.

Reply:
It collapses the children in the left nav by default, rather than opening them.